mephisto/tools/scripts.py

```diff
@@ -234,8 +237,21 @@ def build_custom_bundle(custom_src_dir, run_config: DictConfig):
 
     IGNORE_FOLDERS = {os.path.join(prebuild_path, f) for f in IGNORE_FOLDERS}
     build_path = os.path.join(prebuild_path, "build", "bundle.js")
+
+    if "force_rebuild" not in kwargs:
+        warnings.warn(
+            "Consider adding the force_rebuild property to your hydra config and passing it into build_custom_bundle()\nSteps on how to do this: \nhttps://github.com/facebookresearch/Mephisto/issues/811",
+            stacklevel=2,
+        )
+
+    if "post_install_script" not in kwargs:
+        warnings.warn(
+            "Consider adding the post_install_script property to your hydra config and passing it into build_custom_bundle()\nSteps on how to do this: \nhttps://github.com/facebookresearch/Mephisto/issues/811",
+            stacklevel=2,
+        )
```

Comment:
I think this is too noisy, these flags would only be really used with very limited use cases (developers happening to be working on a library package) yet we would be promoting this warning to all users of mephisto on ever run. I know we discussed adding this during the meeting, but I suggest we just remove it altogether.

Reply:
Agreed on retrospect. Would be worth linking the issue in a comment at the top of the script though.

Reply:
I agree.
I just removed the warnings.
I did add a comment to #811  in case the programmer is looking at the code and is interested in local package development